linters/google-java-format/google-java-format.test.ts

```diff
@@ -0,0 +1,5 @@
+import { linterFmtTest } from "tests";
+
+// TODO(Tyler): Add .clang-format file from configs
```

Comment:
Remove at least line 3. If the current interface is too limiting, https://github.com/trunk-io/plugins/pull/80 offers a bit more.